roles/splunk/tasks/main.yml

```diff
@@ -103,4 +103,7 @@
 # Always set the deployment_task var to your desired task in the playbook! e.g. check_splunk.yml (install|upgrade) or configure_apps.yml  (app/addon install|upgrade)
 - name: Include the specified task file
   tags: always
-  include_tasks: "{{ deployment_task }}"
+  include_tasks: "{{ deployment_task_item }}"
+  loop_control:
+    loop_var: deployment_task_item
+  loop: "{{ deployment_task }}"
```

Comment:
Agreed. There isn't much benefit to this change and it would require all existing playbooks to be updated so it would be very intrusive. I will decline this PR.